src/js/main.js

```diff
@@ -123,6 +126,9 @@ Site.Map = {
     // init pan zones
     _this.setPanZones();
 
+    // Show pattern
+    _this.showPattern();
```

Comment:
I want @mixcas to look at this as I think his intention was to have this stuff happen separately bound to his custom events. I would favor separating the movement and the background changing.